components/rmrk/Gallery/Holder/Holder.vue

```diff
@@ -557,6 +557,8 @@ export default class CommonHolderTable extends mixins(
 }
 </script>
 <style lang="scss">
+@import '@/styles/variables.scss';
```

Comment:
You dont use any variable :)

Reply:
@vikiival `$tablet` is imported from variables

but I would use instead of media query
```css
@include tablet {
  ...
}
```
Bulma ref [here](https://bulma.io/documentation/utilities/responsive-mixins/#named-mixins)